components/collective-page/CollectiveNotificationBar.js

```diff
@@ -69,6 +85,24 @@ const getNotification = (intl, status, collective, host) => {
           description: host ? intl.formatMessage(messages.collectiveCreatedDescription, { host: host.name }) : '',
         };
     }
+  } else if (isEmpty(status) && moneyCanMoveFromEvent(collective)) {
```

Comment:
That would make more sense to me: `get(event, 'type') === CollectiveType.EVENT && moneyCanMoveFromEvent(event)`. Then you don't have to modify `moneyCanMoveFromEvent`

Reply:
Instead of `isEmpty(status)`, can you move that condition to the bottom in last position?